src/Akeneo/Pim/Structure/Bundle/Query/InternalApi/Attribute/SqlCountAttributes.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Structure\Bundle\Query\InternalApi\Attribute;
+
+use Akeneo\Pim\Structure\Component\Query\InternalApi\CountAttributes;
+use Doctrine\DBAL\Connection;
+
+class SqlCountAttributes implements CountAttributes
+{
+    public function __construct(
+        private readonly Connection $connection,
+    ) {
+    }
+
+    public function byFilters(array $filters): int
```

Comment:
I think it's better to use two parameters (includeCodes et excludeCodes) like here https://github.com/akeneo/pim-enterprise-dev/blob/master/components/tailored-export/back/src/Infrastructure/Controller/GetAttributeOptionsAction.php#L45-L46

It will be better readable

```suggestion
    public function byFilters(array $filters): int
```